src/services/collectionService.ts

```diff
@@ -532,6 +532,13 @@ export const deleteCollection = async (
     keepFiles: boolean
 ) => {
     try {
+        if (keepFiles) {
+            const allFiles = await getLocalFiles();
+            const collectionFiles = allFiles.filter((file) => {
+                return file.collectionID === collectionID;
+            });
+            await removeFromCollection(collectionID, collectionFiles);
```

Comment:
A small change, as we will anyways be needing`allFiles` in  `removeFromCollection` let's pass that from here

Reply:
The `removeFromCollection` can then be like

```
export const removeFromCollection = async (
    collectionID: number,
    toRemoveFiles: EnteFile[],
    allFiles?: EnteFile[]
) => {
    try {
        if (!allFiles) {
            allFiles = await getLocalFiles();
        }
        ...
        
 ```
 
 if the caller haven't sent the files then we don't need to read local files